src/screens/AdminPortal/OrganizationTags/OrganizationTags.tsx

```
@@ -223,17 +225,28 @@ function OrganizationTags(): JSX.Element {
 
             <div
               className={styles.subTagsLink}
               data-testid="tagName"
               onClick={() => redirectToSubTags(params.row.id)}
+              onKeyDown={(e) => {
+                if (e.key === 'Enter' || e.key === ' ') {
+                  e.preventDefault();
+                  redirectToSubTags(params.row.id);
+                }
+              }}
+              role="button"
```

Biome (2.3.13)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)